CHANGELOG.md

```diff
@@ -4,6 +4,9 @@
 - Add the +/- icons for the category filtering [#1211](https://github.com/bigcommerce/cornerstone/pull/1211)
 - Fix overlapping logo when using "original" sizing and large logos [#1213](https://github.com/bigcommerce/cornerstone/pull/1213)
 - Fix Product Options hiding Add to Cart on a Google AMP page [#1214](https://github.com/bigcommerce/cornerstone/pull/1214)
+- Fix syntactically incorrect code in carousel and bulk-discount-rates components
+- Fix the possibility of duplicate IDs occurring in product options in certain situations
```

Comment:
Instead of 3 entries. Please move url change into a separate PR. Also just add one changelog entry per PR with link to the PR. So it would look something like this ```
Fix duplicate IDs occurrence in product options in certain situations & syntax fix in carousel and bulk-discount-rates components [#1215](https://github.com/bigcommerce/cornerstone/pull/1215)
```